public/media_engagement/media_engagement.js

```diff
@@ -0,0 +1,276 @@
+//Permorm Authentication then update data download
+firebase.auth().onAuthStateChanged(function(user) {
+    if (user) {
+        console.log("Attempting to bind: " + user.email)
+        const mediadb = firebase.firestore();
+        const settings = { timestampsInSnapshots: true };
+        mediadb.settings(settings);
+        var data = [];
+        mediadb.collection('M&E Dashboard').onSnapshot(res => {
+            // Update data every time it changes in firestore
+            res.docChanges().forEach(change => {
```

Comment:
Print a log message?

Reply:
To fix in pull 18 once moved to production db